m4_sessions/1811_inning/belong_in_tech.md

```diff
@@ -1,30 +1,75 @@
 ## Group Members:
+Justin Stewart
+Jessica Erickson
+Mark Peayer
+Andrew Tobin
 
 
 
 ## Name of Session: 
+Why you belong tech.
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Have you ever felt like an outsider? Is your background not tech related? Come find out why the tech industry really needs you.
 
 ## Session Outcomes 
 
 (What will students learn/leave with?): 
 
-### Recommended Trajectory 
+- Have data and Scientific facts to back diveristy in tech
+- Squash any doubt members belong in tech
+- code switching
+- Being a career changer is ok, new is ok
+
+** Being confident they made the right decision to persue tech.
+** They belong in this community, and nothing disqualifies them.
 
-     Best practices:  No section should be longer than 20 minutes
-     Create a session that YOU would want to attend
-     Share resources in your slack channel
-     All sessions NEED to have a closing activity, reflection, discussion
-     # Introductons 
-     # Intro to new material
-     # Small Group Activity
-     # Closing Discussion
+### Recommended Trajectory 
+     
+ Introduction - and split class into 4 small groups
+ Acticity- 
```

Comment:
This is an awesome opening.  Please add your specific content (either talking points, questions, or even content that you want to cover).  Think about leaving a session that other people could leverage.

Reply:
Will open the floor to get some more understanding to the class. ask why they chose to attend our group, or if they have any fears or doubts themselves.